sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json

```diff
@@ -2597,57 +2331,63 @@
           "type": "integer"
         },
         "inputBlobContainerUri": {
-          "description": "URI containing SAS token to a blob container that contains registry data to sync.",
+          "description": "The URI containing SAS token to a blob container that contains registry data to sync.",
           "type": "string"
         },
         "inputBlobName": {
-          "description": "The blob name to be used when importing from the provided input blob container.",
+          "description": "The blob name to use when importing from the input blob container.",
           "type": "string"
         },
         "outputBlobContainerUri": {
-          "description": "URI containing SAS token to a blob container.  This is used to output the status of the job and the results.",
+          "description": "The SAS token to access the blob container. This is used to output the status and results of the job.",
           "type": "string"
         },
         "outputBlobName": {
-          "description": "The name of the blob that will be created in the provided output blob container.  This blob will contain\r\nthe exported device registry information for the IoT Hub.",
+          "description": "The blob name that will be created in the output blob container. This blob will contain the exported device registry information for the IoT Hub.",
           "type": "string"
         },
         "excludeKeysInExport": {
-          "description": "Optional for export jobs; ignored for other jobs.  Default: false.  If false, authorization keys are included\r\nin export output.  Keys are exported as null otherwise.",
+          "description": "Optional for export jobs; ignored for other jobs. If not specified, the service defaults to false. If false, authorization keys are included in export output. Keys are exported as null otherwise.",
           "type": "boolean"
         },
         "storageAuthenticationType": {
-          "description": "Specifies authentication type being used for connecting to storage account.",
+          "description": "The authentication type used for connecting to the storage account.",
           "enum": [
             "keyBased",
             "identityBased"
           ],
           "type": "string"
         },
         "failureReason": {
-          "description": "System genereated.  Ignored at creation.\r\nIf status == failure, this represents a string containing the reason.",
+          "description": "System genereated.  Ignored at creation. The reason for failure, if a failure occurred.",
+          "type": "string"
+        },
+        "includeConfigurations": {
+          "type": "boolean"
+        },
+        "configurationsBlobName": {
           "type": "string"
         }
       }
     },
     "PurgeMessageQueueResult": {
-      "description": "Result of a device message queue purge operation.",
+      "description": "The result of a device message queue purge operation.",
```

Comment:
fwiw, we're calling this `The result of the purge operation.` in our handwritten client's comments.

Reply:
The concern was that `device message queue purge operation` could be quite a mouthful, and potentially confusing.

Reply:
Put this same description in the internal one.